.eleventy.js

```diff
@@ -163,8 +163,13 @@ module.exports = function (eleventyConfig) {
     return array.slice(0, n);
   });
 
+  const now = new Date();
+  const isDevelopment = process.env.ELEVENTY_ENV === 'development';
+  const livePosts = p => isDevelopment || (p.date <= now && !p.data.draft);
```

Comment:
So, this is also basically a scheduling feature if you build on CI, right?

Could you add a test for it (sufficient to just use a date far in the future)

Reply:
Yes, i'll take care of it!